### PR TITLE
Docs 4827 dh

### PR DESCRIPTION
**Pull request by dhirsch1**
I have moved the Zuora connector user doc into the new template. I see some things that can be improved in terms of information but have not done this since we are only supposed to convert to the new template at this point.

I see that there is a conflict with amazon-dynamodb-connector.adoc. I am not sure how that happened.

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc [hunk 1]
```diff
@@ -207,7 +207,7 @@ XML flow for the DynamoDB create and delete table in Anypoint Studio:
 
 [source,xml,linenums]
 ----
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>ant
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc]: Looks like an ant crept into the code example

File: modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+== XML Namespace
+
+<No information>
+
+== POM File Configuration
+
+To use this connector with Maven, view the pom.xml dependency information in the Dependency Snippets in Anypoint Exchange.
+
+== Keep Session Alive
```
Comment (george-hoffer) [inline modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: Ok to remove this section -- not in the template

File: modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
+= Zuora - XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== XML Namespace
+
+<No information>
```
Comment (george-hoffer) [inline modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: Put these statements in a source block for the XML Namespace section:

http://www.mulesoft.org/schema/mule/zuora-connector
http://www.mulesoft.org/schema/mule/zuora-connector/current/mule-zuora-connector.xsd

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc]: What caused this?

File: modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: Do you mean the Keep Session Alive section?

File: modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: These URLs are not found when I put them into the browser.

Comment (dhirsch1) [reply modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: OK

**Pull request closed without merging**